examples/CMakeLists.txt

```diff
@@ -65,12 +74,32 @@ endif()
 
 add_example(server_ctt server_ctt.c)
 
+if (WIN32)
+    install(PROGRAMS ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/server_ctt.exe
+            DESTINATION bin
+            RENAME ua_server_ctt.exe)
+else()
+    install(PROGRAMS ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/server_ctt
+            DESTINATION bin
+            RENAME ua_server_ctt)
+endif()
```

Comment:
I guess this if can be substituted with: 
~~~cmake
install(PROGRAMS  $<TARGET_FILE:server_ctt>
        DESTINATION bin
        RENAME ua_server_ctt)
~~~
but to be honest I did not try it.

Reply:
Ah, cool! Will try that!
Thanks